src/service/url-replacements-impl.js

```diff
@@ -426,7 +430,7 @@ export class UrlReplacements {
    * @template T
    */
   getAccessValue_(getter, expr) {
-    return this.getAccessService_().then(accessService => {
+    return this.getAccessService_(this.win_).then(accessService => {
```

Comment:
Lets do in follow up.

On Fri, Sep 9, 2016 at 3:38 PM, Dima Voytenko <notifications@github.com>
wrote:

> In src/service/url-replacements-impl.js
> <https://github.com/ampproject/amphtml/pull/4908#discussion_r78259462>:
>
> > @@ -426,7 +430,7 @@ export class UrlReplacements {
> >     * @template T
> >     */
> >    getAccessValue_(getter, expr) {
> > -    return this.getAccessService_().then(accessService => {
> > +    return this.getAccessService_(this.win_).then(accessService => {
>
> Would it just be easier to bind in constructor? It will also ensure that
> we don't pass a wrong @this to that method.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4908/files/9dc6e433e9326d421095feb7f85465681e43f296#r78259462>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT9PYFAvt6w6T8lox7D0HZs6HPlP5ks5qod_ygaJpZM4J5hjb>
> .
>